app/components/Form/legoForm.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import { reduxForm, SubmissionError } from 'redux-form';
+import { handleSubmissionError } from './utils';
+import { pick } from 'lodash';
+type Props = {
+  onSubmitFail: any => any,
+  onSubmit: any => Promise<*>,
+  /* Enable auto submissionError
+   * This only works if 'onSubmit' is passed into legoForm,
+   * and not if it is passed into 'handleSubmit()'*/
+  enableSubmissionError?: boolean,
+  /* Move the screen to the first error in the list on SubmissionError */
+  enableFocusOnError?: boolean,
+  /* Also pick these values based on the keys given on the array
+   * Should be used when there is no visible filed for the value,
```

Comment:
"no visible filed for the value"?

Reply:
I guess filed = field. This comment should probably be rewritten to better document exactly what `enableValuePicking` and `pickAdditionalValues` do.